bindings/pydeck-carto/examples/scripts/carto_layer_geo_query.py

```diff
@@ -26,4 +31,4 @@
 view_state = pdk.ViewState(latitude=0, longitude=0, zoom=1)
 
 r = pdk.Deck(layer, map_style=pdk.map_styles.ROAD, initial_view_state=view_state)
-r.to_html("carto_layer_geo_query.html", open_browser=True)
+r.to_html(join(dirname(__file__), "carto_layer_geo_query.html"))
```

Comment:
Added this line (here and in other examples) to ensure example output is written to the same folder as the example itself. This makes it easier to run all the example scripts together (new make command: `make test-scripts`) and verify the output (`npx serve examples/scripts` or similar).